src/flags/index.js

```diff
@@ -16,7 +16,11 @@ export type Flag <T> = {
   default?: ?() => (Promise<?T> | ?T),
   required?: ?boolean,
   optional?: ?boolean,
-  parse?: ?(?string, ?Command<*>, string) => (Promise<?T> | ?T)
+  parse?: ?(?string, ?Command<*>, string) => (Promise<?T> | ?T),
+  completions: ?{
```

Comment:
I feel this should be singular: `flags.app.completion.options` rather than `flags.app.completions.options`